akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -1176,7 +1176,8 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
 
   def receiveGet(key: KeyR, consistency: ReadConsistency, req: Option[Any]): Unit = {
     val localValue = getData(key.id)
-    log.debug("Received Get for key [{}], local data [{}]", key, localValue)
+    if (log.isDebugEnabled)
+      log.debug("Received Get for key [{}], local data [{}]", key, localValue.map(_.data.getClass.getName))
```

Comment:
The class is not very interesting here, so let's make this 
`"Received Get for key [{}]"`